core/ui/public/i18n/en/translation.json

```diff
@@ -673,7 +673,9 @@
         "url_placeholder": "E.g. http://127.0.0.1:2000",
         "node_helper": "Select the node where the route will be created",
         "skip_cert_verify": "Skip certificate validation",
-        "skipCertVerify_tooltip": "Skip certificate validation for the backend target URL"
+        "skipCertVerify_tooltip": "Skip certificate validation for the backend target URL",
+        "instance_pattern": "Allowed characters: A-Z, a-z, 0-9, _ and -",
```

Comment:
This string is not used, it can be removed

Reply:
instance_pattern is the string sent by the json schema, we can remove it or keep it as you want ?

Reply:
Let's use `instance_pattern` instead of `bad_characters` string (see previous comment)